odoo/addons/base/models/res_lang.py

```diff
@@ -249,6 +249,25 @@ def _lang_code_to_urlcode(self, code):
                 return urlc
         return self._lang_get(code).url_code
 
+    @api.model
+    def _get_enabled_lang_code(self):
+        """ Get the list of code enabled (aka installed and default) """
+        langs = self.get_enabled()
+        return [lang[0] for lang in langs]
+
+    @api.model
+    @tools.ormcache()
+    def get_enabled(self):
```

Comment:
Enabled is strange (I would think those are activate or installed)... it is more "get available for usage in a many2one when used to know someone's language", but this could be a long method name... mmmh ...